hbp_knowledge/receptors/buckingham2009.bel

```diff
@@ -1276,22 +1379,25 @@ by their ability of cells to reduce 3-(4,5-dimethylthiazol-
 impaired when alpha7 was silenced by RNAi, suggesting
 that Abeta may exert its toxicity, at least in part, through
 a pathway that includes alpha7 nAChRs (Qi et al., 2007)."
-
+SET Confidence = "High"
 SET CellLine="SH-SY5Y"
 
 complex(p(HGNC:CHRNA7),a(CHEBI:"amyloid-beta")) -> path(HBP:neurotoxicity)
 
 UNSET CellLine
+UNSET Confidence
 
 SET Evidence="It is
 noteworthy that this internalization was blocked by
 alpha-bungarotoxin, which may indicate that alpha-bungarotoxin
 either inhibits binding of Abeta to the alpha7 receptor
 (and therefore that Abeta toxicity results from binding of
 Abeta to alpha7 nAChRs) or directly inhibits alpha7 nAChR internalization."
-
+SET Confidence = "Low"
 a(MESH:Bungarotoxins) -| act(a(CHEBI:"amyloid-beta"))
+#the evidence says alpha7 internalization and not amyloid-beta so should the above line be changed with the former?
```

Comment:
The sentence before says : "The toxic action of Aβ may involve α7-dependent internalization of amyloid", so i would stick to this statment as it is